src/msg/producer/writer/message_pool_test.go

```diff
@@ -60,15 +57,4 @@ func TestMessagePool(t *testing.T) {
 	m.Close()
 	require.Nil(t, m.pb.Value)
 	p.Put(m)
-
-	m = p.Get()
```

Comment:
Mind giving a quick tl;dr on why these assertions are no longer relevant? Not sure I get it (yet).